go.mod

```diff
@@ -32,3 +32,5 @@ require (
 	gopkg.in/yaml.v2 v2.2.2
 	gotest.tools v2.2.0+incompatible // indirect
 )
+
+go 1.13
```

Comment:
Is it auto added by the tool? If 1.13 is required, make sure you update Dockerfile to 1.13 as well.